installation.php

```diff
@@ -43,7 +43,8 @@
                     <div class="text-center">
                         <div id="creating-a-usb-choices" class="column linked">
                             <a class="button creating-a-usb-on-windows" href="#creating-a-usb-on-windows">Windows</a>
-                            <a class="button creating-a-usb-on-others" href="#creating-a-usb-on-others">OS X & Ubuntu</a>
+                            <a class="button creating-a-usb-on-osx" href="#creating-a-usb-on-others">OS X</a>
```

Comment:
I think emersion means why have you changed the class and not the href for OS X, and why is it still `*-others` for the Ubuntu section.